docs/source/java/quickstartguide.rst

```diff
@@ -0,0 +1,482 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+.. default-domain:: java
+.. highlight:: java
+
+.. _java_quickstartguide:
+
+=================
+Quick Start Guide
+=================
+
+.. contents::
+
+Arrow java manage data in Vector Schema Root (somewhat analogous to tables and record
+batches in the other Arrow implementations). Before to create a Vector Schema Root let's
+define another topics neededs for that purpose.
+
+Create A Value Vector
+*********************
+
+It's called `array` in the columnar format specification. Represent a one-dimensional
```

Comment:
reST uses double backticks. But usually for this we should use quotes. `Also known as "arrays" in the columnar format.`

Reply:
Added